data/sql/updates/pending_db_world/rev_1626863078498353400.sql

```diff
@@ -1,12 +1,12 @@
 INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626863078498353400');
 
 -- Changed the id of the spell so they use the Sneak that puts them on stealth. Changed the comments to know what spell are they using
-UPDATE `smart_scripts` SET `action_param1` = 22766, `comment` = 'Jadefire Shadowstalker - On Respawn - Cast Sneak (Stealth mode)' WHERE (`entryorguid` = 7110) AND (`source_type` = 0) AND (`id` = 0);
-UPDATE `smart_scripts` SET `comment` = 'Jadefire Shadowstalker - On Aggro - Cast Jadefire' WHERE (`entryorguid` = 7110) AND (`source_type` = 0) AND (`id` = 1);
-UPDATE `smart_scripts` SET `comment` = 'Jadefire Shadowstalker - On Aggro - Cast Slowing Poison' WHERE (`entryorguid` = 7110) AND (`source_type` = 0) AND (`id` = 2);
+UPDATE `smart_scripts` SET `action_param1` = 22766, `comment` = 'Jadefire Shadowstalker - On Respawn - Cast Sneak (id:22766) (Stealth mode)' WHERE (`entryorguid` = 7110) AND (`source_type` = 0) AND (`id` = 0);
```

Comment:
-- Cast Sneak (id:2276) instead of Sneak (id:11013) to actually enter stealth. Sniffed on Classic
-- Cast Slowing Poison (id:7992) instead of Slowing Poison (id:14897). Sniffed on Classic 
-- Add both spell name and id to descriptions

Reply:
Changed the comments